src/ontodia/viewUtils/connectionsMenu.tsx

```diff
@@ -467,15 +605,19 @@ class ConnectionsList extends React.Component<ConnectionsListProps, {}> {
     };
 
     render() {
-        const links = this.getLinks();
+        const isSmartMode = this.isSmartMode;
+
+        const links = !isSmartMode ? this.getLinks() : [];
```

Comment:
Swap the `then` and `else` branches of the ternary operator to remove negation from the condition.